_posts/modules/2019-03-15-apps.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: "Luos apps"
+categories: 2_modules
+desc: What is an App.
+order: 1
+wip: 1
+---
+{% include var.md %}
+<div class="wip_img"></div>
+<blockquote class="warning"><strong>Work in progress</strong><br /><br />We are working on this page...</blockquote><br />
+
+This page details what is an app, how to create and initialize one, and how to use it.
+
+# What is an app?
+An applications or app is a type of module. An app is a module which only manages software items such a functions, and which can communicate with and use any other module ([drivers](drivers) or apps). An app can't have access to hardware like a driver can.
+Apps can be placed in any nodes on a Luos network without any hardware or code modifications. However, the choice of the hosting node can impact global performances of the system.
+
+> **Note:** Theoretically, we separate the definitions of an app and a driver: they are both modules but they don't have the same functions into a Luos architecture. However, it's practically possible to create a module which can be in the same time an app and a driver, that is, a module which controls hardware and can have access to any other module.
```

Comment:
```suggestion
> **Note:** Theoretically, we separate the definitions of an app and a driver: they are both modules but they don't have the same functions into a Luos architecture. However, it's practically possible to create a module which can be in the same time an app and a driver, that is, a module which controls hardware and take control of any other driver or app. We advice to avoid as you can to mix them. A driver should be dumb and passiv an App act.
```

Reply:
My text is not really good. feel free to remake it.